src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -83,6 +83,16 @@ def __init__(
             new_config["steps_offset"] = 1
             scheduler._internal_dict = FrozenDict(new_config)
 
+        if safety_checker is None:
+            logger.warn(
+                f"You have disabed the safety checker for {self.__class__} by passing `safety_checker=None`.Please"
+                " make sure you have very good reasons for this and have considered the consequences of doing so.The"
+                " `diffusers` team does not recommend disabling the safety under ANY circumstances and strongly"
+                " suggests to not disable the `safety_checker` by NOT passing `safety_checker=None` to"
```

Comment:
I don't think the second "by NOT passing `safety_checker=None` to `from_pretrained`." is necessary